Model/Behavior/MongoBehavior.php

```diff
@@ -11,6 +11,11 @@
 class MongoBehavior extends ModelBehavior {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class "MongoBehavior" has 269 lines, which is greater than the 200 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2042)